crates/core/src/network/connection.rs

```diff
@@ -222,10 +221,10 @@ impl Connection {
         ConnectThrottle::note_connect_status(&destination, result.is_ok());
         match result {
             Err(ref e) => {
-                info!(%direction, "Couldn't connect to {}: {}", destination, e);
+                debug!(%direction, "Couldn't connect to {}: {}", destination, e);
             }
             Ok((_, task_id)) => {
-                info!(%direction, %task_id, "Connection established to {}", destination);
+                debug!(%direction, %task_id, "Connection established to {}", destination);
```

Comment:
Can we keep this one info, I agree that connection failure can be reduced, but this one shouldn't be too chatty unless we have large number of nodes.

Reply:
Maybe with your latest changes to reduce the number of connections between nodes this is less of a problem. In some of the test failures I was analyzing, it seemed that a good amount of logging came from creating the connections. Then again, how important is this information for the average user when not debugging a problem?

Reply:
it's not important, but it's the only signal we have to show that we have successfully observed/connected to a node. I'm okay with reducing it to debug and increase the level if we need to.